pallets/accelerated-bn-crypto/src/utils.rs

```diff
@@ -0,0 +1,85 @@
+// Copyright 2024, Horizen Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Generic executions of the operations for *Arkworks* elliptic curves.
+
+// As not all functions are used by each elliptic curve and some elliptic
+// curve may be excluded by the build we resort to `#[allow(unused)]` to
+// suppress the expected warning.
+
+#![allow(dead_code)]
+
+use crate::ArkScale;
+use ark_ec::AffineRepr;
+use ark_scale::hazmat::ArkScaleProjective;
+use ark_serialize::{CanonicalSerialize, Compress};
+use ark_std::{test_rng, vec, vec::Vec, UniformRand};
+
+pub type ScalarFieldFor<AffineT> = <AffineT as AffineRepr>::ScalarField;
+
```

Comment:
Since `ScalarFieldFor` is only used in `lib.rs`, you could move the definition there. Then, in `lib.rs`, the `utils` mod (which is only used in tests and benchmarks) can be put behind `#[cfg(any(test, feature = "runtime-benchmarks"))]`